src/FsAutoComplete/CodeFixes/ResolveNamespace.fs

```diff
@@ -8,9 +8,29 @@ open FsAutoComplete.LspHelpers
 open FsAutoComplete
 open FSharp.Compiler.Text
 open FSharp.Compiler.EditorServices
+open System.Text.RegularExpressions
 
 type LineText = string
 
+let undefinedName =
+  [ "not define"
+    "nedefinuje|Není definovaný|Není definované|Není definovaná|Nemáte definovaný"
+    "definiert nicht|nicht.*? definiert"
+    "no define|no está definido|no está definida"
+    "ne définit|n'est pas défini"
+    "non definisce|non è definito|non è definita"
+    "定義(され|し)ていません"
+    "정의(하지 않|되지 않았|되어 있지 않)습니다"
+    "nie definiuje|Nie zdefiniowano|nie jest zdefiniowany"
+    "não define|não está definido"
+    "не определяет|не определено|не определены|не определен"
+    "tanımlamıyor|tanımlı değil"
+    "未.*?定义"
+    "未定義" ]
```

Comment:
Probably not for this PR to solve but yeah @baronfel this I can see this getting unwieldy.

Reply:
This was my primary hesitation - I had hoped that we could find a way to detect these issues in another way.